src/deck/drivers/src/usddeck.c

```diff
@@ -365,17 +372,63 @@ static void usddeckEventtriggerCallback(const eventtrigger *event)
   if (logBufferReadPtr < logBufferWritePtr + dataSize)
   {
     /* write data into buffer */
-    uint16_t event_id = eventtriggerGetId(event);
+    uint16_t event_id = cfg->eventId;
     memcpy(logBufferWritePtr, &event_id, sizeof(event_id));
     logBufferWritePtr += sizeof(event_id);
     memcpy(logBufferWritePtr, &ticks, 4);
     logBufferWritePtr += 4;
-    memcpy(logBufferWritePtr, event->payload, event->payloadSize);
-    logBufferWritePtr += event->payloadSize;
+    if (payloadSize) {
+      memcpy(logBufferWritePtr, payload, payloadSize);
+      logBufferWritePtr += payloadSize;
+    }
+
+    for (int i = 0; i < cfg->numVars; ++i)
+    {
+      logVarId_t varid = cfg->varIds[i];
+      switch (logGetType(varid))
+      {
+      case LOG_UINT8:
+      case LOG_INT8:
+      {
+        memcpy(logBufferWritePtr, logGetAddress(varid), sizeof(uint8_t));
+        logBufferWritePtr += sizeof(uint8_t);
+        break;
+      }
```

Comment:
There is no variable block in this case, is the new scope needed?

Could this not be just:
```
      case LOG_INT8:
        memcpy(logBufferWritePtr, logGetAddress(varid), sizeof(uint8_t));
        logBufferWritePtr += sizeof(uint8_t);
        break;
```
Same for cases below.

Reply:
Should be fixed.